lib/emailer.js

```diff
@@ -63,7 +63,17 @@ module.exports.sendEmailValidationEmail = async function(toEmail, token, tokenEx
   }
 }
 
-module.exports.sendConfCreatedEmail = async function(toEmail, phoneNumber, pin, durationInMinutes, freeAt, confUrl) {
+module.exports.sendConfCreatedEmail = async function(toEmail, phoneNumber, pin, durationInMinutes, freeAt, confUrl, pollUrl) {
+  var pollHtml = ""
+  if(pollUrl) {  
+    pollHtml = `<hr />
+    <div class="paragraph">
+    <div>PS : Donnez-nous votre avis sur le service : </div>
```

Comment:
"Donnez-nous votre avis sur le service :" (sans le PS et le double ":")

Reply:
tu peux faire des suggestions directement
```suggestion
    <div>Donnez-nous votre avis sur le service :</div>
```